### ISSUE TITLE
How to use early stopping for saving the best train accuracy?

### ISSUE THREAD

**Issue by hoangthangta** [JayYip/m3tl#102]
Hello, I am looking for a way to save the best train accuracy. The only thing I saw in the training process is this line:

```
model = train_bert_multitask(
        problem = problem,
        num_epochs = EPOCHS,
        problem_type_dict = problem_type_dict,
        processing_fn_dict = processing_fn_dict,
        continue_training = False,
        params = params 
    )
```

How to intervene to the training loop?

**Comment by JayYip** [comment]
You should use `callbacks`.
